File: src/lessons/turtleart/DiskFourEntity.c
//RemoteTurtle

void quadrant();

/* BEGIN TEMPLATE */
void run() {
	/* BEGIN SOLUTION */
	quadrant();
	setColor(red);
	quadrant();
	setColor(orange);
	quadrant();
	setColor(magenta);
	quadrant();
}
void quadrant() {
	int i;
	for (i=0; i<90;i++) {
		forward(100);
		backward(100);
		right(1);
	}
	/* END SOLUTION */
}
/* END TEMPLATE */

File: resources/exercises/turtleart/diskfour/DiskFourEntity.py
# BEGIN SOLUTION
def quadrant():
	for i in range(90):
		forward(100);
		backward(100);
		right(1);

quadrant();
setColor(Color.RED);
quadrant();
setColor(Color.ORANGE);
quadrant();
setColor(Color.MAGENTA);
quadrant();
# END SOLUTION
